### DISCUSSION TITLE
@aws-cdk/aws-apigatewayv2-integrations-alpha: Lambda async invocation docs

### DISCUSSION THREAD

**Discussion by vmashnitskaya** [aws/aws-cdk#28111] [General]
### Describe the issue

Hello! I am using the following examples to create the API Gateway + Lambda https://docs.aws.amazon.com/cdk/api/v2/docs/aws-apigatewayv2-alpha-readme.html. Currently, I would like to use async invocation described here https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-integration-async.html. But there is no documentation on this. I tried to use parameter mapping https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-parameter-mapping.html but the `X-Amz-Invocation-Type` should be reserved value and is not applied. Could you please add the examples for such case?

### Links

-

**Comment by pahud** [comment]
Thank you. As this is a request for general guidance I am moving this to discussion.

**Reply by vmashnitskaya** [reply]
Thank you! And can I receive the guidance on this issue?

**Reply by vmashnitskaya** [reply]
Hello @pahud , is there any update on this discussion?

**Comment by vmashnitskaya** [comment]
Hello! Could someone please help me with this?

**Comment by vmashnitskaya** [comment]
Hello! Could someone please help me with this?
